src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb

```diff
@@ -139,21 +139,21 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             MyBase.SendCommitUniqueCompletionListItem(Sub(a, n, c) handler.ExecuteCommand(a, n, c), Sub() Return)
         End Sub
 
-        Public Overrides Async Function AssertNoCompletionSession() As Task
-            Await WaitForAsynchronousOperationsAsync()
```

Comment:
Do you mean that 
1. AssertNoAsynchronousOperationsRunning() is not useful in the modern implementation of the completion.
2. You want to keep checks with AssertNoAsynchronousOperationsRunning() in case if another API would require it in the future
3. We can modify AssertNoAsynchronousOperationsRunning and WaitForAsynchronousOperationsAsync for the modern completion but keep them in tests.

Please correct my understanding of the idea.

---
In reply to: [314785331](https://github.com/dotnet/roslyn/pull/38021#discussion_r314785331) [](ancestors = 314785331)